test/deprecated-keyword/messageFiltering.chpl

```diff
@@ -171,6 +171,11 @@ var x1102 = 1102;
 deprecated "Lorem ipsum :proc:`!abc` dolor sit amet"
```

Comment:
It looks like this case already existed - I think that means your cases should be added before line 167 and this line should be removed